decidim-core/app/models/decidim/taxonomy_filter.rb

```diff
@@ -3,6 +3,12 @@
 module Decidim
   # Represents the association between a taxonomy and a filterable entity within the system.
   class TaxonomyFilter < ApplicationRecord
+    include Decidim::TranslatableResource
+    include Decidim::Traceable
+    include Decidim::SanitizeHelper
```

Comment:
I cannot say that i like this approach. Also this is not met nowhere else. The least thing you could do is to create a presenter, move this to that presenter.